waku/v2/rpc/relay_test.go

```diff
@@ -14,7 +19,11 @@ func makeRelayService(t *testing.T) *RelayService {
 	require.NoError(t, err)
 	err = n.Start()
 	require.NoError(t, err)
-	return &RelayService{n}
+	return &RelayService{
+		n,
+		make(map[string][]*pb.WakuMessage),
```

Comment:
If we end up closing `ch` in `Stop()` then `ch` should be initialized in `Start`, so it's possible to restart the RelayService by calling `Start` again  (maybe we'll add an rpc method or api function to start/stop services and protocols on demand)

Reply:
I added the init of ch in start